misc/triage/labels.yaml

```diff
@@ -0,0 +1,142 @@
+---
+repo: aquasecurity/trivy
+labels:
+
+# kind
+- name: kind/api-change
+  color: e11d21
+  description: Categorizes issue or PR as related to adding, removing, or otherwise changing an API
+- name: kind/bug
+  color: e11d21
+  from: bug
+  description: Categorizes issue or PR as related to a bug.
+- name: kind/cleanup
+  color: c7def8
+  description: Categorizes issue or PR as related to cleaning up code, process, or technical debt.
+- name: kind/deprecation
+  color: e11d21
+  from: enhancement
+  description: Categorizes issue or PR as related to a feature/enhancement marked for deprecation.
+- name: kind/documentation
+  color: c7def8
+  description: Categorizes issue or PR as related to documentation.
+- name: kind/feature
+  color: c7def8
+  description: Categorizes issue or PR as related to a new feature.
+- name: kind/testing
+  color: c7def8
+  description: Categorizes issue or PR as related to a unit/integration test.
+- name: kind/regression
+  color: e11d21
+  description: Categorizes issue or PR as related to a regression from a prior release.
+- name: kind/security
+  color: f4dddc
+  description: security issues
+
+# lifecycle
+- name: lifecycle/active
+  color: 8fc951
+  description: Indicates that an issue or PR is actively being worked on by a contributor.
```

Comment:
I may be missing something but it seems like there is some overlap between the lifecycle stages and the triage stages. 

What is the lifecycle state for an issue that we intend to work on, but we're not actively working on yet (e.g. one that is in `priority/important-soon` and not yet being worked on)? How about the lifecycle state for an issue that is waiting for more information? 

What triggers this `lifecycle/active` label to be added - do contributors to mark this label when they start work on something? Or could this be implied by having an `Assigned` person on the issue?

Reply:
You're right. It seems unnecessary. If an issue with `priotiry/` label, we don't have to add `lifecycle/active`. Also, all issues should have `priority/` label. In short, `lifecycle/active` will be never used. I removed it.
https://github.com/aquasecurity/trivy/pull/473/commits/c4375911a8a62423422d5248e70b268a352c3504

What do you think about `lifecycle/frozen`? This is a label that tells the bot not to auto-close. It might not be necessary, but it may be necessary in very rare cases. For example, we got a bug report, which is difficult to fix shortly. It looks affecting only small users, but we're not sure. The issue should not be closed due to staleness so that we can watch the reaction from users. But we may be able to use `priority/awaiting-more-evidence` instead. If so, we can remove `lifecycle/frozen` as well.